pallets/staking/src/lib.rs

```diff
@@ -2644,6 +2646,32 @@ decl_module! {
                     .map(|p| p.intended_count = new_intended_count)
             })
         }
+
+
+        /// Declare no desire to either validate or nominate.
+        ///
+        /// Effects will be felt at the beginning of the next era.
+        ///
+        /// The dispatch origin for this call must be a GC.
+        /// And, it can be only called when [`EraElectionStatus`] is `Closed`.
+        #[weight = <T as Config>::WeightInfo::chill_from_governance()]
+        pub fn chill_from_governance(origin, identity: IdentityId, stash_keys: Vec<T::AccountId>) {
```

Comment:
Anytime a `Vec` is a parameter, it's length needs to be used in the weight function and benchmarks.
```suggestion
        #[weight = <T as Config>::WeightInfo::chill_from_governance(stash_keys.len())]
        pub fn chill_from_governance(origin, identity: IdentityId, stash_keys: Vec<T::AccountId>) {
```